File: C++/LogicPrograms/primenumber.cpp
#include <iostream>
using namespace std;

int main(){
    int num,i,count=0;
   cout<<"Enter the number for checking the primenumber\n";
   cin>>num;
   for(i=1;i<=num;i++){
   if(num%i==0){
   count++;
   }
}
 if(count==2){
    cout<<"It is primenumber "<<num;
   }else{
   cout<<"its not prime number"<<num;
   }
   }

File: Core-Java Practice/Programs/src/Logical/PrimeNumber.java
package Logical;

import java.util.Iterator;
import java.util.Scanner;

public class PrimeNumber {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		System.out.println("Enter the number>> ");

		int num = scanner.nextInt();
		int count = 0;
		for (int i = 2; i <= num; i++) {
			if (num % i == 0) {
				count++;

			}
			if (count == 0) {
				System.out.println(num + "Prime Number");
			} else {
				System.out.println(num + "Not Prime Number");

			}

		}

	}

}
